CONTRIBUTING.md

```diff
@@ -1,106 +1,157 @@
 # Contributing
 
-Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+Hi there!
+We're thrilled that you'd like to contribute to this project.
+Your help is essential for keeping it great.
 
 Contributions to this project are [released](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) to the public under the [project's open source license](LICENSE).
 
-This project adheres to the [Contributor Covenant Code of Conduct](http://contributor-covenant.org/). By participating, you are expected to uphold this code.
+This project adheres to the [Contributor Covenant Code of Conduct](http://contributor-covenant.org/).
+By participating, you are expected to uphold this code.
 
 The majority of contributions won't need to touch any Ruby code at all.
 
 ## Getting started
 
-Before you can start contributing to Linguist, you'll need to set up your environment first. Clone the repo and run `script/bootstrap` to install its dependencies.
+Before you can start contributing to Linguist, you'll need to set up your environment first.
+Clone the repo and run `script/bootstrap` to install its dependencies.
 
-    git clone https://github.com/github/linguist.git
-    cd linguist/
-    script/bootstrap
+```console
+$ git clone https://github.com/github/linguist.git
```

Comment:
I suggest you don't add the `$` here. It makes copy-pasting bits of code much harder. Especially with multiple lines of shell commands. Users must be able to copy-paste directly to their shell.

Reply:
@crisluengo That's an issue that's easily solved on GitHub's side. See github/markup#1223.

Reply:
@Alhadis I agree with you, but in meantime, it is not solved and very annoying for users.

Reply:
Alright, fixed. 👍